documentation/wiki/Labels.md

```diff
@@ -0,0 +1,12 @@
+# MSBuild Labels
+Here's a brief explanation on the labels most often used by the MSBuild team excluding obvious ones such as `bug`.
```

Comment:
```suggestion
Here's a brief explanation on the labels most often used by the MSBuild team excluding hopefully-self-evident ones such as `bug`.
```

"Obvious" is a dangerous word for public writing: if it wasn't obvious to the reader they feel bad.